### PR TITLE
Fix legacy PID Tuning Sliders

### PR DESCRIPTION
**Pull request by haslinghuis**
After #1821 PID Tuning done in software (API 1.43 or below) - now called legacy - was not working anymore.

### CODE REVIEW

**Comment by haslinghuis** [comment]
Tested with software sliders (API 1.43) and firmware sliders (API 1.44)

**Review by McGiverGim — APPROVED** [review]
Approved. Thanks for the fix.
One of the "requisites" for the first tuning sliders was to let out all the `semver` conditions to make it simpler. The decisions about what to show or not, or what to do was based only in the parameters passed to them. With the evolution of the file we forgot this and we add a lot of them.
But this is a legacy one so I think we can simply maintain it.

**Review by mikeller — APPROVED** [review]

**Comment by Asizon** [comment]
@haslinghuis after that, firmware sliders on 4.3 doesnt work anymore :(

**Comment by mikeller** [comment]
@haslinghuis: Let me know if you have time to look into fixing this - if not I think it is best to revert this for now.

File: src/js/TuningSliders.js [hunk 1]
```diff
@@ -368,24 +391,27 @@ TuningSliders.calculateNewPids = function() {
...
+        FC.TUNING_SLIDERS.slider_dterm_filter = TuningSliders.sliderDTermFilter ? 1 : 0;
+        FC.TUNING_SLIDERS.slider_dterm_filter_multiplier = TuningSliders.sliderDTermFilterMultiplier * 100;
+
+        FC.TUNING_SLIDERS.slider_gyro_filter = TuningSliders.sliderGyroFilter ? 1 : 0;
+        FC.TUNING_SLIDERS.slider_gyro_filter_multiplier = TuningSliders.sliderGyroFilterMultiplier * 100;
+
+        Promise.resolve(true)
+        .then(() => { return MSP.promise(MSPCodes.MSP_SET_TUNING_SLIDERS, mspHelper.crunch(MSPCodes.MSP_SET_TUNING_SLIDERS)); })
+        .then(() => { return MSP.send_message(MSPCodes.MSP_SET_PID); })
+        .then(() => { return MSP.send_message(MSPCodes.MSP_SET_PID_ADVANCED); });
```
Comment (klutvott123) [inline src/js/TuningSliders.js:413]: These were changed from ```MSP_PID```/```MSP_PID_ADVANCED``` to ```MSP_SET_PID``` /```MSP_SET_PID_ADVANCED```. I think this is what breaks it.

Comment (haslinghuis) [reply src/js/TuningSliders.js:413]: @klutvott123 Yes! Thanks. Will make a patch later this day.

Comment (mikeller) [reply src/js/TuningSliders.js:413]: Well spotted @klutvott123!

**Pull request merged**